### PR TITLE
fix(Tabs): Fix styled

### PR DESCRIPTION
**Pull request by BennettStaley**
Fixes styled exports on the tab component so it exports the styled
versions of everything.

### CODE REVIEW

**Comment by BennettStaley** [comment]
I didn't deploy this as its no style change, just some export changes.

File: src/components/Tabs/Tab.js [hunk 1]
```diff
@@ -39,16 +39,16 @@ const TabComponent = ({ children, className, Icon, index, onClick }) => (
...
 );
 
-export const Tab = styled(TabComponent)`
+export const Tab = styled(TabComponent) `
   opacity: ${props => (props.selected ? 1 : 0.7)};
   height: ${props => (props.Icon && props.children ? 72 : 48)}px;
 `;
 
-const TabContentContainer = styled.div`
+const TabContentComponent = ({ children, className }) => (
```
Comment (hsarb) [inline src/components/Tabs/Tab.js]: this revets the new API styling we were trying to accomplish

Comment (BennettStaley) [reply src/components/Tabs/Tab.js]: Unless I'm mistaken, its impossible to style that element reliably without calling styled on it. As the component that was exported wasn't a SC, but its child was.

Comment (hsarb) [reply src/components/Tabs/Tab.js]: So would it make sense then just to export the styled div then?

```
export const TabContent = styled.div`
  padding: 24px;
  width: 100%;
`;
```

Comment (BennettStaley) [reply src/components/Tabs/Tab.js]: Actually... yes? I think so.

**Pull request merged**